examples/sveltekit-pwa/src/app.html

```diff
@@ -11,6 +11,6 @@
 		%sveltekit.head%
 	</head>
 	<body>
-		<div id="svelte">%sveltekit.body%</div>
+		<div>%sveltekit.body%</div>
```

Comment:
commenting here since it will only let me comment on changed lines...  line 6 and 8 could be changed to `%sveltekit.assets%/favicon.svg`

Reply:
do we need also to use it on the examples? rn I'm using `/manifest.webmanifest`.

Reply:
> commenting here since it will only let me comment on changed lines... line 6 and 8 could be changed to `%sveltekit.assets%/favicon.svg`

done

Reply:
Yes, I think it's a good idea to use with `manifest.webmanifest` too